resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/scheduler/VolcanoFeatureStep.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.deploy.k8s.features.scheduler
```

Comment:
Would it be a good idea to add a new Maven/Sbt module, e.g. `kubernetes/volcano` next to `kubernetes/core` for all Volcano specific classes ?

Reply:
If this suggestion is approved then you will have to revert the usage of `${classOf[VolcanoFeatureStep].getName}` in the setting's documentation :-/

Reply:
IMO, I prefer to stay this featurestep in here, because it's more lite and simple. It also does not introduce more complex dependencies or something. So, a separate module may not be necessary.